e2e/utils/element-helper.ts

```diff
@@ -0,0 +1,58 @@
+import { ElementFinder, Key, element, by, ElementArrayFinder } from 'protractor';
+
+export class ElementHelper {
+  async clear(elem: ElementFinder, length?: number) {
+    const inputText = await elem.getAttribute('value');
+    length = length || inputText.length || 100;
+    let backspaceSeries = '';
+    for (let i = 0; i < length; i++) {
+      backspaceSeries += Key.BACK_SPACE;
+    }
+    elem.sendKeys(backspaceSeries);
```

Comment:
Wouldn't it be better to clear it and then send one backspace?

Reply:
that's the case here that `.clear()` not always work, solution for it is to send series of backspace.